src/components/Conversation/index.tsx

```diff
@@ -120,6 +122,31 @@ function Wrap(props: HTMLAttributes<HTMLDivElement>) {
     )
 }
 
+function TokenGatedBox() {
+    const isPromoting = useIsDelegatedAccountBeingPromoted()
+
+    const usePromote = useTokenGatedPromoteDelegatedAccount()
+
+    const promote = () => {
+        // somehow, add the tokenId
```

Comment:
Been giving this some thought and am still not happy with any of the options. The dreamy way would be to fetch the NFT ids owned by the user and compare that against the accepted ones on the tokenGate, but the ERC721 standard doesn't allow for that out-of-the-box. Managed to get a workaround working using events, but that won't fly while using public RPC endpoints because most make you batch your event fetches.

Thinking now that the best alternative would be to ask the user to manually input the token ID they desire to use to join with via an input